src/router/routes.js

```diff
@@ -12,4 +12,6 @@ export default [
 	{ path: '/checkout-vue', component: () => import('@/pages/Checkout/CheckoutPage') },
 	{ path: '/checkout', component: () => import('@/pages/Checkout/CheckoutBetaPage') },
 	{ path: '/protocol', component: () => import('@/pages/Protocol') },
+	{ path: '/build-vue', component: () => import('@/pages/Build/BuildPage') },
+	{ path: '/build-vue/docs', component: () => import('@/pages/Build/DocsPage') },
```

Comment:
I think you can place the docs component route into a 'children' array to the /build-vue object.
https://github.com/vuejs/vue-router/blob/dev/examples/nested-routes/app.js
* Note the addition of the Default component which may or may not be required...have to test that out.